README.md

```diff
@@ -11,6 +11,7 @@ Tyk Pump is a pluggable analytics purger to move Analytics generated by your Tyk
 - ElasticSearch (2.0+)
 - Graylog
 - InfluxDB
+- InfluxDB2
```

Comment:
@flyte can you add a separated section specifying this pump is especially useful for InfluxDB 2.x and InfluxDB 1.8+?

Reply:
I'm actually quite new to InfluxDB so I don't really know how 1.8 and 2.1 differ. As far as I'm aware, 1.8 uses username/password instead of tokens and the "bucket" is called a "database". If that is the case then it makes the config for this pump incompatible.

I don't really have the time available to research this distinction fully at the moment I'm afraid.